source/features/hide-inactive-deployments.js

```diff
@@ -0,0 +1,22 @@
+import select from 'select-dom';
+import observeEl from '../libs/simplified-element-observer';
+
+export default function () {
+	observeEl('.js-discussion', () => {
+		const deployments = select.all('.discussion-item .deployment-meta');
+		deployments.pop(); // Don't hide the last deployment, even if it is inactive
+
+		for (const deployment of deployments) {
+			const isInactiveDeployment = select.exists('.is-inactive', deployment);
+			if (isInactiveDeployment) {
```

Comment:
This can be `if select.exists` without the extra var. Same for discussionItem on the next line